packages/table-react/src/ExpandableTableRowCell.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import cx from "classnames";
+import { ExpandButton } from "@fremtind/jkl-expand-button-react";
+import { TableCell, TableCellProps } from ".";
+
+interface ExpandableTableRowCellProps extends TableCellProps {
+    isOpen?: boolean;
```

Comment:
På denne og `onClick` hadde det vært nice med en kommentar på at den sendes inn av `ExpandableTableRow`
```suggestion
    /** Settes automatisk av ExpandableTableRow */
    isOpen?: boolean;
```